models/Voting.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose')
+
+const schema = new mongoose.Schema(
+  {
+    user_id: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'User',
+      required: true,
+    },
+    first_movie_id: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'Movie',
+      required: true,
+    },
+    second_movie_id: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'Movie',
+      required: true,
+    },
+    isVoted: {
+      type: mongoose.Schema.Types.Boolean,
+      required: true,
+    },
+    first_movie_won: {
+      type: mongoose.Schema.Types.Boolean,
+      required: true,
+    },
```

Comment:
I would consider a three-value-type like (0 | 1 | 2) or (null | true | false) or (null | "first" | "second"). What do you think?